### PR TITLE
Fuzzer

### PR DESCRIPTION
**Pull request by ikarienator**

### CODE REVIEW

**Comment by michaelficarra** [comment]
These should be 4 separate PRs.

**Comment by ikarienator** [comment]
ping @michaelficarra

File: src/main/java/com/shapesecurity/shift/fuzzer/Fuzzer.java [hunk 1]
```diff
@@ -0,0 +1,974 @@
...
+    };
+  }
+
+  @NotNull
+  private static <T> Gen<List<T>> many(@NotNull Gen<T> gen) {
+    return many(MANY_BOUND, gen);
+  }
+
+  @NotNull
+  private static <T> Gen<NonEmptyList<T>> many1(@NotNull Gen<T> gen) {
```
Comment (michaelficarra) [inline src/main/java/com/shapesecurity/shift/fuzzer/Fuzzer.java]: Really, you need to de-dupe this somehow.

Comment (ikarienator) [inline src/main/java/com/shapesecurity/shift/fuzzer/Fuzzer.java]: Done.

**Pull request merged**